projects/github-actions/repo-gardening/src/tasks/gather-support-references/index.js

```diff
@@ -0,0 +1,165 @@
+const debug = require( '../../debug' );
+
+/* global GitHub, WebhookPayloadIssue */
+
+/**
+ * Search for a previous comment from this task in our issue.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - Issue number.
+ * @returns {Promise<number>} Promise resolving to a number.
+ */
+async function getListComment( octokit, owner, repo, number ) {
+	let commentID = 0;
+
+	debug( `gather-support-references: Looking for a previous comment from this task in our issue.` );
+
+	for await ( const response of octokit.paginate.iterator( octokit.rest.issues.listComments, {
+		owner,
+		repo,
+		issue_number: +number,
```

Comment:
Let's avoid unnecessary pagination.
```suggestion
		issue_number: +number,
		per_page: 100,
```